clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1179,6 +1186,50 @@ protected String getConfigValue(ConfigKey key, String headerName) {
         }
     }
 
+    private static String niceMemoryUnits(ConfigKey key) {
+        final long defaultValueBytes = ((Number) key.defaultValue).longValue();
+        long defaultValue = defaultValueBytes;
+        int i = 0;
+        while (i < 5) {
+            if (defaultValue % 1024 != 0) {
+                break;
+            }
+            defaultValue /= 1024;
+            i++;
+        }
+        switch (i) {
+            case 1:
+                return " (=" + defaultValue + "KiB)";
```

Comment:
The prefix for kilo, is lowercase `k`

Reply:
You're right, but in this case we're dealing with kibibytes, not kilobyes, and [it seems](https://en.wikipedia.org/wiki/Kibibyte) in this case KiB is the standard abbreviation.